inc/saiport.h

```diff
@@ -1566,6 +1547,16 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_PRBS_RX_STATUS,
 
+    /**
+     * @brief Attribute data for #SAI_PORT_ATTR_PRBS_RX_STATE
+     * Used for clear on read status/count register.
+     * Adapter should return return SAI_STATUS_NOT_SUPPORTED if not supported.
```

Comment:
typo: return return

Reply:
> Hi Jai,
> 
> We need the below understanding to be documented:
> 
> There are multiple SAI attributes/stat-id (viz., SAI_PORT_ATTR_PRBS_RX_STATE, SAI_PORT_ATTR_PRBS_RX_STATUS, SAI_PORT_PRBS_RX_STATUS_LOCK_WITH_ERRORS) for PORT PRBS status. It depends on the underlying ASIC as to which of these attributes are applicable. The suggestion was that the application would first query SAI_PORT_ATTR_PRBS_RX_STATE. If the SAI implementation returns that the attribute is unsupported, the application would then fallback to query the existing SAI_PORT_ATTR_PRBS_RX_STATUS and SAI_PORT_PRBS_RX_STATUS_LOCK_WITH_ERRORS.

Comments in header file are limited in characters.
We will add this detail in release notes

Reply:
> typo: return return
Fixed
https://github.com/opencomputeproject/SAI/pull/1179/commits/e276ffafbc70c59d7171dfe4ce5e22b90d25ddfa